docs/howto/request-public-listing.md

```diff
@@ -0,0 +1,80 @@
+(request-public-listing)=
+# Make your charm discoverable
+
+> See first: [](#charm-maturity)
```

Comment:
Something's missing here. (But this comment might not be relevant anymore if you make the other changes I suggest.)

Reply:
It auto-fills the title from the ref. But I could make it explicit and that's probably nicer when reading the Markdown rather than generated cos.